### ISSUE TITLE
type 'Color' is not a subtype of type 'bool'

### ISSUE THREAD

**Issue by Paul-Todd** [yako-dev/flutter-status-alert#2]
I had to fix this by putting brackets around the ternary operator - not sure but it looks like the ternary operator has higher precedence than the ?? operator causing this error

```
widget.subtitleOptions.style.color ?? Theme.of(context).brightness == Brightness.light ? lightAccent : darkAccent
```

Should read

```
widget.subtitleOptions.style.color ?? (Theme.of(context).brightness == Brightness.light ? lightAccent : darkAccent)
```

Notice the brackets added to the ternary operator to fix the issue

**Comment by Z3rolive** [comment]
No fixes yet. Can't change color in any of the color property.

**Comment by yadaniyil** [comment]
Fixed in 0.1.2

**Comment by Octet83** [comment]
I'm using 0.1.2 and it doesn't work, same error as above
